### DISCUSSION TITLE
Does it support debugging async functions?

### DISCUSSION THREAD

**Discussion by acking-you** [SeaQL/FireDBG.for.Rust#16] [Q&A] [answered]
As the title suggests, I'm curious to know if there is support for debugging async functions. As a rust newbie, I'm curious if there's a better way to debug async functions other than logging 😢

**Comment by tyt2y3** [comment, marked as answer]
I agree with you! Indeed debugging async Rust was our original motivation in creating FireDBG, but the pre-requisite is to first tackle sync Rust. We are still fighting with some low level async runtime details, and hopefully we'll be able to release it soon.

Please star our repo and stay tuned!

**Reply by Milo123459** [reply]
Is there a tracking issue?

**Reply by tyt2y3** [reply]
> Is there a tracking issue?

let me create one and post the progress there

**Reply by Milo123459** [reply]
Awesome!
